src/App.jsx

```diff
@@ -14,33 +15,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABET = {
+  id: 1,
+  title: 'Sort alphabetically',
+  class: 'is-info',
+};
+const SORT_BY_LENGTH = {
+  id: 2,
+  title: 'Sort by length',
+  class: 'is-success',
+};
+const REVERSE = {
+  id: 3,
+  title: 'Reverse',
+  class: 'is-warning',
+};
+const RESET = {
+  id: 4,
+  title: 'Reset',
+  class: 'is-danger',
+};
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [sortBtn, setSortBtn] = useState('');
+  const [sortedGoods, setSortedGoods] = useState([...goodsFromServer]);
```

Comment:
You don't need this state. just create a variable for sorted goods it will be recalculated after each time you switch sort method since component will be rerendered